.travis.yml

```diff
@@ -13,6 +13,8 @@ env:
 
 
 before_install:
+    # Proof-of-concept XQuartz installation
+    - brew cask install xquartz
```

Comment:
> I believe "casks" installed by "brew cask" are left alone by any other brew subcommands that aren't "brew cask ___".

I don't know to be honest. (Not a Mac user.)

> This means that the "brew" command may still be installed.

That sounds right.

> Also, it seems as though XQuartz may have been re-added to the Travis CI build image... which would make all of this unnecessary.

:+1: